README.md

```diff
@@ -79,7 +79,7 @@ System Requirements
 
 Turi Create requires:
 
-* Python 2.7, 3.5, 3.6
+* Python 2.7, 3.5, 3.6, 3.7
```

Comment:
Where are we documenting the interaction with macOS/Linux?

Reply:
We're not yet. I'll add that.

Reply:
I've add this.